pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -336,6 +342,19 @@ func (o *RoleModificationOptions) AddRole() error {
 	if err != nil {
 		return err
 	}
+
+	if len(o.RoleBindingName) > 0 {
+		filteredRoleBindings := []*authorizationapi.RoleBinding{}
```

Comment:
filtering the list to a single item is weird... it also means if the named rolebinding already exists but points to a different roleref, it'll get missed here... that should be an error